pages/home/visualizations/commit_metrics.py

```diff
@@ -0,0 +1,228 @@
+from dash import html, dcc
+import dash_bootstrap_components as dbc
+from dash import callback
+from dash.dependencies import Input, Output, State
+from db_manager.augur_manager import AugurManager
+
+
+commit_total = dbc.Card(
```

Comment:
For all of the metric files: Do we need more documentation throughout (I know this is my code lol)? If so we should open an issue and I can work on that after this PR is merged

Reply:
yeah let's do that